qchem/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# Release 0.18.0-dev
```

Comment:
Not sure what to do with an empty changelog. Should we just skip the 0.18.0 entry?